nwb_conversion_tools/datainterfaces/ecephys/tutorial/sortingtutorialdatainterface.py

```diff
@@ -54,6 +55,7 @@ def __init__(self, duration: float = 10.0, num_units: int = 10, sampling_frequen
             self.sorting_extractor.set_unit_property(
                 unit_id=unit_id, property_name="custom_unit_column", value="A custom value"
             )
+        self.writer_class = map_si_object_to_writer(self.sorting_extractor)(self.sorting_extractor)
```

Comment:
I have a question about this kind of assignations.
Here the function maps the self.sorting_extractor to the appropriate writer class and then we instantiate in the spot. 
Would it not be better to have that in two lines intead for clarity?
```python
WriterClass = map_si_object_to_writer(self.sorting_extractor)
self.writer_class = WriterClass(self.sorting_extractor)

Just curious about whether would improve readability
```

Reply:
I'd second this; using double non-nested parentheses on a single line is a very odd look. There are other parts of the code around the place that take these exact lines and break them up almost exactly how @h-mayorquin suggests.

The only question is whether to use CamelCase; I do believe it is returning an instance of a writer class, not an auto-built class definition, so it would use the lower-underscore instead of CamelCase